File: allFiles/inputFiles/valid3.py
# nested if-else

a = 10                                                                                                                                                                  
b = 20                                                                                                                                                                  
c = 15                                                                                                                                                                  
if a == b and b == c:                                                                                                                                        
        print("All are Equal", )                                                                                                                                        
elif a > b:                                                                                                                                                             
    if a > c:                                                                                                                                                           
        print("a is the largest", )                                                                                                                                     
    else:                                                                                                                                                               
        print("c is the largest", )                                                                                                                                     
else:                                                                                                                                                                   
    if b > c:                                                                                                                                                           
        print("b is the largest", )                                                                                                                                     
    else:                                                                                                                                                               
        print("c is the largest", )

File: allFiles/outputFiles/valid3.cpp
#include<iostream>
using namespace std;
#include<string.h>
int main()
{
        int a_int,b_int,c_int;

        a_int = 10;
        b_int = 20;
        c_int = 15;
        if ( a_int == b_int && b_int == c_int )
        {
                cout <<  "All are Equal" << endl;
        }
        else if ( a_int > b_int )
        {
                if ( a_int > c_int  )
                {
                        cout <<  "a is the largest" << endl;
                }
                else
                {
                        cout <<  "c is the largest" << endl;
        }
                }
        else
        {
                if ( b_int > c_int  )
                {
                        cout <<  "b is the largest" << endl;
                }
                else
                {
                        cout <<  "c is the largest" << endl;
                }
        }

}
